parm/MOM_input_template_025

```diff
@@ -82,11 +82,11 @@ WRITE_GEOM = 2                  ! default = 1
 TRIPOLAR_N = True               !   [Boolean] default = False
                                 ! Use tripolar connectivity at the northern edge of the
                                 ! domain.  With TRIPOLAR_N, NIGLOBAL must be even.
-NIGLOBAL = 1440                 !
+NIGLOBAL = NX_GLB               !
```

Comment:
Eventually I was hoping that all the MOM_input templates could be collapsed into one file, even for waves. It may not be possible but that was where I was going w/ this.

Reply:
Agreed, I didn't see a way to do waves vs no waves without just adding an extra template but would be happy if there was a way to do this.  I think there are even more differences between 1/2 and 1/4 than waves and no waves.